frontend/pages/group/data.vue

```diff
@@ -44,6 +44,9 @@ export default defineComponent({
       foods: i18n.tc("general.foods"),
       units: i18n.tc("general.units"),
       labels: i18n.tc("data-pages.labels.labels"),
+      categories: i18n.tc("category.categories"),
```

Comment:
Is there a way to add dividers or headers to this list? Now that it's as long as it is, it's a bit hard to mentally parse.
I don't know if we can, just a thought

![image](https://github.com/mealie-recipes/mealie/assets/71845777/ea8a79c6-8122-4f8f-9570-90d127985792)

Reply:
yeah, was feeling the same, but the BaseOverflowButton did not support that. 

added the divider MenuItem book to extend the BaseOverflowButton functionality. Please take a look at it if it looks fine to you.